src/main/index.js

```diff
@@ -186,14 +185,69 @@ const init = $context => {
       e.preventDefault()
       const id = $(this).data('content-id')
       const href = $(this).attr('href')
-      convertWithAjax(id, href, $target => {
-        Common.trigger('new context', $target)
-        $('.convert-button').hide()
+      $.ajax({
+        url: href,
+        type: 'GET',
+        async: true,
+        beforeSend: () => {
+          $('#loading').show()
+        },
+        complete: () => {
+          $('#loading').hide()
+        }
+      }).done(function(data) {
+        function getEditedArticle($all) {
+          const $editable = $all.find(
+            `.editable[data-id="${id}"][data-edit-type="ory"]`
+          )
+          return $editable.closest('article').length
+            ? $editable.closest('article')
+            : $all.find('#content-layout article')
+        }
+
+        const $target = getEditedArticle($('body'))
+        const $dataArticle = getEditedArticle($(data))
+        $target.html($dataArticle.html())
+      })
```

Comment:
This ajax call is here twice. Is this intentional or a mistake?

Reply:
Nope, thanks. And I also saw the `$('#loading').hide()` which I can just move down and fix the mentioned issue in the comment :)